packages/nc-gui-v2/components/dashboard/settings/UIAcl.vue

```diff
@@ -0,0 +1,146 @@
+<script setup lang="ts">
+import { useToast } from 'vue-toastification'
+import { viewIcons } from '~/utils/viewUtils'
+import { h, useNuxtApp, useProject } from '#imports'
+import MdiReload from '~icons/mdi/reload'
+import MdiContentSave from '~icons/mdi/content-save'
+import MdiMagnify from '~icons/mdi/magnify'
+
+const { $api, $e } = useNuxtApp()
+const { project } = useProject()
+const toast = useToast()
+
+const roles = $ref<Array<string>>(['editor', 'commenter', 'viewer'])
+let isLoading = $ref(false)
+let tables = $ref<Array<any>>([])
+let searchInput = $ref<string>('')
```

Comment:
unnecessary generic `<string>` - will already be inferred by having `''` in the ref.